src/coreComponents/constitutive/solid/DamageSpectral.hpp

```diff
@@ -246,15 +288,15 @@ class DamageSpectralUpdates : public DamageUpdates< UPDATE_BASE >
     smallStrainUpdate( k, q, strainIncrement, stress, stiffness.m_c );
   }
 
-
+  ///accessor for strain energy density - in this case, it only cares about the positive part
   GEOSX_HOST_DEVICE
   virtual real64 getStrainEnergyDensity( localIndex const k,
                                          localIndex const q ) const override final
   {
     return m_strainEnergyDensity( k, q );
   }
 
-
+  ///accessor for the energy threshold, which, in the cohesive formulation, is independent of the regularization length
```

Comment:
```suggestion
/**
  * @brief accessor for the energy threshold, which, in the cohesive formulation, is independent of the regularization length
  * return the critical energy threshold value.
  */
```

Reply:
is this because the single-line comment is too long?